frontend/packages/dev-console/src/components/form-utils/form-utils-types.ts

```diff
@@ -0,0 +1,10 @@
+export interface FormFooterProps {
+  type?: 'button' | 'submit' | 'reset';
```

Comment:
You don't need this prop plus the name is very confusing. And what will reset type do? You are not using this in reset button

Reply:
Removed it